src/packages/core/section/section-views/section-views.element.ts

```diff
@@ -141,16 +142,17 @@ export class UmbSectionViewsElement extends UmbLitElement {
 		return this._dashboards.length > 0
 			? html`
 					<uui-tab-group slot="header" id="dashboards">
-						${this._dashboards.map(
-							(dashboard) => html`
+						${this._dashboards.map((dashboard) => {
+							const dashboardName = dashboard.meta.label ?? dashboard.name;
+							const dashboardPath =
+								'dashboard/' + dashboard.meta.pathname ? dashboard.meta.pathname : umbracoPath(dashboardName);
```

Comment:
I think `umbracoPath()` is used for our msw handlers to prepend the management API path to those, so wouldn't the path look like: `/umbraco/api/v1/.....` if they didn't provide a fallback?